lib/i18n/backend/active_record.rb

```diff
@@ -45,6 +45,33 @@ def store_translations(locale, data, options = {})
           end
         end
 
+        def reload!
+          @initialized = false
+          @translations = nil
+        end
+
+        def initialized?
+          @initialized ||= false
+        end
+
+        def init_translations
+          @translations = Translation.all.each.with_object({}) do |t, memo|
```

Comment:
Could you please move `Translation.all.each.with_object({}) do |t, memo| ...` to [the `Translation` class](https://github.com/svenfuchs/i18n-active_record/blob/master/lib/i18n/backend/active_record/translation.rb#L57), for example to `.to_hash` method

Reply:
Much cleaner, thank you for the input.